### PR TITLE
Updated build version

### PR DESCRIPTION
**Pull request by bennsimon**

### CODE REVIEW

File: opensrp-giz-malawi/src/main/java/org/smartregister/giz/repository/GizMalawiRepository.java [hunk 1]
```diff
@@ -96,6 +98,9 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
                 case 7:
                     upgradeToVersion7OutOfArea(db);
                     break;
+                case 8:
+                    upgradeToVersion16AddServiceGroupColumn(db);
```
Comment (dubdabasoduba) [inline opensrp-giz-malawi/src/main/java/org/smartregister/giz/repository/GizMalawiRepository.java]: This is not updating to version 16.  Please rename this appropriately

File: opensrp-giz-malawi/src/main/java/org/smartregister/giz/repository/GizMalawiRepository.java [hunk 2]
```diff
@@ -176,9 +181,24 @@ private void runLegacyUpgrades(SQLiteDatabase database) {
         upgradeToVersion13(database);
```
Comment (dubdabasoduba) [inline opensrp-giz-malawi/src/main/java/org/smartregister/giz/repository/GizMalawiRepository.java]: Please rename this methods correctly. GIZ's DB is at a max of 8 I don't see how you would be upgrading to 13

File: opensrp-giz-malawi/src/main/java/org/smartregister/giz/repository/GizMalawiRepository.java [hunk 3]
```diff
@@ -176,9 +181,24 @@ private void runLegacyUpgrades(SQLiteDatabase database) {
...
         upgradeToVersion14(database);
```
Comment (dubdabasoduba) [inline opensrp-giz-malawi/src/main/java/org/smartregister/giz/repository/GizMalawiRepository.java]: Please rename this methods correctly. GIZ's DB is at a max of 8 I don't see how you would be upgrading to 14

File: opensrp-giz-malawi/src/main/java/org/smartregister/giz/repository/GizMalawiRepository.java [hunk 4]
```diff
@@ -176,9 +181,24 @@ private void runLegacyUpgrades(SQLiteDatabase database) {
...
         upgradeToVersion15RemoveUnnecessaryTables(database);
```
Comment (dubdabasoduba) [inline opensrp-giz-malawi/src/main/java/org/smartregister/giz/repository/GizMalawiRepository.java]: Please rename this methods correctly. GIZ's DB is at a max of 8 I don't see how you would be upgrading to 15

File: opensrp-giz-malawi/src/main/java/org/smartregister/giz/repository/GizMalawiRepository.java [hunk 5]
```diff
@@ -176,9 +181,24 @@ private void runLegacyUpgrades(SQLiteDatabase database) {
...
+        upgradeToVersion16AddServiceGroupColumn(database);
```
Comment (dubdabasoduba) [inline opensrp-giz-malawi/src/main/java/org/smartregister/giz/repository/GizMalawiRepository.java]: Please rename this methods correctly. GIZ's DB is at a max of 8 I don't see how you would be upgrading to 16

File: opensrp-giz-malawi/src/main/java/org/smartregister/giz/repository/GizMalawiRepository.java (hunk 1 above)
Comment (bennsimon) [reply opensrp-giz-malawi/src/main/java/org/smartregister/giz/repository/GizMalawiRepository.java]: ok

File: opensrp-giz-malawi/src/main/java/org/smartregister/giz/repository/GizMalawiRepository.java (hunk 2 above)
Comment (bennsimon) [reply opensrp-giz-malawi/src/main/java/org/smartregister/giz/repository/GizMalawiRepository.java]: ok

**Pull request merged**